datafusion/core/src/datasource/physical_plan/parquet/row_group_filter.rs

```diff
@@ -487,11 +487,23 @@ mod tests {
         let schema_descr = get_test_schema_descr(vec![field]);
         let rgm1 = get_row_group_meta_data(
             &schema_descr,
-            vec![ParquetStatistics::int32(Some(1), Some(10), None, 0, false)],
+            vec![ParquetStatistics::int32(
+                Some(1),
+                Some(10),
+                None,
+                Some(0),
```

Comment:
Due to https://github.com/apache/arrow-rs/pull/6216 (where null counts is now Option)